plugins/web/opentelemetry-instrumentation-document-load/src/documentLoad.ts

```diff
@@ -115,6 +115,10 @@ export class DocumentLoadInstrumentation extends InstrumentationBase<unknown> {
         }
       });
 
+      rootSpan.setAttribute(HttpAttribute.HTTP_URL, location.href);
+      rootSpan.setAttribute(HttpAttribute.HTTP_USER_AGENT, navigator.userAgent);
+      rootSpan.setAttribute(AttributeNames.PAGE_TITLE, document.title);
```

Comment:
Not sure about this one... this could veer into PII or other sensitive info.  We generally take pains not to collect things like this out of the box (e.g., the use of xpaths rather than labels/text for clicked elements in `user-interaction`.

Reply:
Got it. I removed the page title.

Reply:
I agree with page title it might contain the user name or name and surname etc.  With regards to `location.href` I think this is too much information it will also produce also very high cardinality. On the other hand the page load in majority of cases in SPA will be connected with the same address, unless you have old style page where each subpage is loaded. But then probably some mechanism for lowering the cardinality should be added similar it is done in express / http or for xmlhttprequest (we only save host). Can you please tell the real use case why do think it should be useful ?

Reply:
Right now I don't actually know which pages of a website is slow or loads more resources than others. I think it's a case for static websites where URL is predictable and different websites can produce significantly different traces (other resources, scripts, load times). In case of SPA applications there can occur high cardinality like random hashes in URLs but documentLoad spans will be generated only when page opens.
In XHR instrumentation I see the requested url is directly saved as the `http.url` attribute so there can occur the same problem (common case with adding timestamp as query param to prevent caching). So is preventing high cardinality in span attributes very crucial?

Reply:
In span attributes data can be high cardinality. In names it is low cardinality. You are correct `http.url` is already used in other areas. I think it is pretty well accepted these days too that URL is not private or PII typically.

Reply:
I'm fine with `location.href` in concept - Daniel's right on all counts, though I would point out that we (Splunk) apply it to *all* spans in our product and you might consider doing that too (e.g., in a `SpanProcessor`).  It does seem odd to have it just on one bit of instrumentation.

Reply:
@johnbley That's an interesting idea. Is it also called `http.url`? Did you think about proposing it to the standard (maybe `@opentelemetry/web`?). When I think about it, adding `location.href` by default makes a lot of sense.